tests/Lexer.test.cpp

```diff
@@ -138,4 +138,23 @@ TEST_CASE("lookahead")
     CHECK_EQ(lexer.lookahead().type, Lexeme::Eof);
 }
 
+TEST_CASE("stringInterpolation")
```

Comment:
Definitely need more tests here to account for error cases, like that `` `foo{{bad}}bar` `` case.